metals/src/main/scala/scala/meta/internal/parsing/Trees.scala

```diff
@@ -103,6 +101,9 @@ final class Trees(
           case Parsed.Success(tree) =>
             trees(path) = tree
             List()
+          case _ =>
+            scribe.info("Received unexpected parser result")
```

Comment:
Will this happen often? Should this be info or warn?

Reply:
I changed severity to the `error` and I logged the class's simple name. Maybe it's too cautious?